frontend/app/src/main/java/com/example/recipez/JSONArrayAdapter.java

```diff
@@ -36,7 +36,7 @@ public JSONArrayAdapter(@NonNull Context context, int resource, @NonNull ArrayLi
         mContext = context;
         mResource = resource;
         allObjects = new ArrayList<>(objects);
-        objectType = objectType;
+        this.objectType = objectType;
```

Comment:
Needed to add "this" keyword so it would assign it to the object's objectType variable; before it was assigning it to nothing

Reply:
Thanks for catching this 🦖